testing/src/main/graphql/betterbotz/schema.json

```diff
@@ -5054,6 +5054,48 @@
               },
               "isDeprecated": false,
               "deprecationReason": null
+            },
+            {
+                "name": "_int_function",
```

Comment:
I had trouble with modeling the `https://github.com/stargate/stargate/pull/901/files#diff-cf660cd57cf1a5b7f88859dec98db42f352e3daa32a50ac012966a29c57e005eR13` in this `schema.json`.
@olim7t Is this what we want to achieve? (the current schema compiles properly)